tracing.cabal

```diff
@@ -14,8 +14,7 @@ maintainer:         mtth@apache.org
 copyright:          2020 Matthieu Monsch
 build-type:         Simple
 extra-source-files: README.md
-tested-with:
-  GHC ==8.10.7 || ==9.0.2 || ==9.2.8 || ==9.4.8 || ==9.6.4 || ==9.8.2
+tested-with:        GHC == { 8.10.7, 9.0.2, 9.2.8, 9.4.8, 9.6.4, 9.8.2 }
```

Comment:
You can do the same with `tracing-effectful.cabal` for consistency :)

Reply:
Done !